synapse/storage/databases/main/schema/full_schemas/54/full.sql.sqlite

```diff
@@ -67,11 +67,6 @@ CREATE TABLE IF NOT EXISTS "user_threepids" ( user_id TEXT NOT NULL, medium TEXT
 CREATE INDEX user_threepids_user_id ON user_threepids(user_id);
 CREATE VIRTUAL TABLE event_search USING fts4 ( event_id, room_id, sender, key, value )
 /* event_search(event_id,room_id,sender,"key",value) */;
-CREATE TABLE IF NOT EXISTS 'event_search_content'(docid INTEGER PRIMARY KEY, 'c0event_id', 'c1room_id', 'c2sender', 'c3key', 'c4value');
```

Comment:
Are these tables used anymore? I can't find any references to them and these are the tables causing `sqlite3.OperationalError: object name reserved for internal use: event_search_content` errors.

Reply:
This was clarified by @richvdh in https://github.com/matrix-org/synapse/issues/8996#issuecomment-754271226. These are shadow tables from the fts4 virtual table above.